lib/ramble/docs/configuration_files.rst

```diff
@@ -561,3 +561,21 @@ format is as follows:
 
 Variables can be defined as lists, scalars, or can refer to a variable defined in
 another fully qualified experiment (through the ``cross_ref_var`` syntax).
+
+-------------------
+Experiment Repeats:
+-------------------
+
+The repeats config options within the ``config`` section are used to define a number
+of times each experiment will be repeated. Summary statistics will be calculated for
+the set of repeats. Its format is as follows:
+
+.. code-block:: yaml
+
+    ramble:
```

Comment:
nit; Can you remove `ramble;` here? In the config section it's intended to start with `config:` while in the workspace config, you'd use `ramble:config` instead.

Reply:
Done